lib/matplotlib/_api/__init__.py

```diff
@@ -1,4 +1,16 @@
+"""
+Helper functions for managing the Matplotlib API.
+
+.. warning:
+
+    This module and its submodules are for internal use only.
```

Comment:
```suggestion
    This module and its submodules are for internal use only.  We may change
    the API at any time with no warning.  If you want to use these functions in your
    own code please vendor them.
```

Reply:
I've chosen a slightly different expression. In particular I've left out the vendoring part.  First, I'm not sure that less experienced developers know that term. But more importantly, I don't think we should/need to actively promote the possibility of vendoring. A bit philosophical: Bringing up the possibility would already imply that it *could* be useful for other people, but I'd rather not make *any* statement about external usage of this code. Users can themselves decide if they want to copy and adapt the code (n.b. some messages explicitly contain "Matplotlib").